cdc/processor/manager.go

```diff
@@ -33,14 +33,14 @@ import (
 type commandTp int
 
 const (
-	commandTpUnknow commandTp = iota //nolint:varcheck,deadcode
+	commandTpUnknown commandTp = iota //nolint:varcheck,deadcode
```

Comment:
I think the abbreviation Tp here is confusing too.

Reply:
me too, let's change it to `commandType`